### PR TITLE
Bug/32935 past events color

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/32935

### CODE REVIEW

File: src/views/month/single-event.php [hunk 1]
```diff
@@ -193,6 +193,6 @@
 ?>
 
 <div id="tribe-events-event-<?php echo esc_attr( $event_id ); ?>" class="<?php tribe_events_event_classes() ?>" data-tribejson='<?php echo esc_attr( tribe_events_template_data( $post ) ); ?>'>
-	<h3 class="tribe-events-month-event-title"><a href="<?php echo esc_url( $link ) ?>" class="url"><?php echo $title ?></a></h3>
+	<h3 class="tribe-events-month-event-title"><span><a href="<?php echo esc_url( $link ) ?>" class="url"><?php echo $title ?></a></span></h3>
```
Comment (zbtirrell) [inline src/views/month/single-event.php]: Was this extra span necessary, or could this style have been applied directly to the `<a>`?

Comment (trishasalas) [inline src/views/month/single-event.php]: It was necessary.  I first tired to apply the text decoration to the link but a previously declared `text-decoration=none` had a higher specificity.  I also tried using a pseudo element but that was just messy! 

It is admittedly a bit hacky and I'm not in love withe the code although I do (personally) like the visual it creates.  

There are enough other a11y issues and this is minor enough that if the span is a problem I don't mind revisiting this in a larger audit/scope.

Comment (trishasalas) [inline src/views/month/single-event.php]: ...or maybe I can remove the span and apply the line-through to the <h3> tag instead ;p

**Comment by barryhughes** [comment]
* Not a big problem but we should avoid committing assets that haven't been changed (I see one `.css` file that changed but many `.min.css` files)
* Should the change made in `css/tribe-events-theme.css` have been applied to `postcss/tribe-events-theme.pcss` (and then be compiled _back_ to vanilla CSS)? 
* Do we need a matching rule in `postcss/tribe-events-full.pcss`?

**Comment by barryhughes** [comment]
:+1:

**Pull request merged**